src/stan-users-guide/user-functions.Rmd

```diff
@@ -114,37 +115,52 @@ quantities block.
 
 ### Type declarations for functions {-}
 
-| Function Argument    | Local            | Block                                 |
-|:---------------------|:-----------------|:--------------------------------------|
-| (unsized)            | (unconstrained)  | (constrained)                         |
-| `int`                | `int`            | `int`                                 |
-|                      |                  | `int<lower = L>`                      |
-|                      |                  | `int<upper = U>`                      |
-|                      |                  | `int<lower = L, upper = U>`           |
-| `real`               | `real`           | `real`                                |
-|                      |                  | `real<lower = L>`                     |
-|                      |                  | `real<upper = U>`                     |
-|                      |                  | `real<lower = L, upper = U>`          |
-| `vector`             | `vector[N]`      | `vector[N]`                           |
-|                      |                  | `vector[N]<lower = L>`                |
-|                      |                  | `vector[N]<upper = U>`                |
-|                      |                  | `vector[N]<lower = L, upper = U>`     |
-|                      |                  | `ordered[N]`                          |
-|                      |                  | `positive_ordered[N]`                 |
-|                      |                  | `simplex[N]`                          |
-|                      |                  | `unit_vector[N]`                      |
-| `row_vector`         | `row_vector[N]`  | `row_vector[N]`                       |
-|                      |                  | `row_vector[N]<lower = L>`            |
-|                      |                  | `row_vector[N]<upper = U>`            |
-|                      |                  | `row_vector[N]<lower = L, upper = U>` |
-| `matrix`             | `matrix[M, N]`   | `matrix[M, N]`                        |
-|                      |                  | `matrix[M, N]<lower = L>`             |
-|                      |                  | `matrix[M, N]<upper = U>`             |
-|                      |                  | `matrix[M, N]<lower = L, upper = U>`  |
-|                      | `matrix[K, K]`   | `corr_matrix[K]`                      |
-|                      | `matrix[K, K]`   | `cov_matrix[K]`                       |
-|                      | `matrix[K, K]`   | `cholesky_factor_corr[K]`             |
-|                      | `matrix[K, K]`   | `cholesky_factor_cov[K]`              |
+| Function Argument | Local           | Block                                   |
```

Comment:
Isn't this a repeat of another table earlier?  Or vice versa?  Can we have one just reference the other or should it all be repeated?

Reply:
This table was actually the original, which I copied into the reference manual. I think if we only want it one place, it's better suited there. I can also move the explanation over to that chapter if we desire.